pkg/mcp/msi/filesystem.go

```diff
@@ -35,6 +35,7 @@ type ListDirectoryResultEntry struct {
 
 type ListDirectoryResult struct {
 	Entries []ListDirectoryResultEntry `json:"entries" jsonschema:"The directory content entries."`
+	Logs    []string                   `json:"logs,omitempty" jsonschema:"Logs related to path translation and other operations."`
```

Comment:
Can we use https://modelcontextprotocol.io/specification/2025-11-25/basic#meta in https://pkg.go.dev/github.com/modelcontextprotocol/go-sdk@v1.1.0/mcp#CallToolResult ?